code/WorkInProgress/rewardsLocker.dm

```diff
@@ -1179,3 +1179,63 @@
 					src.player.claimed_rewards.Add(S.type)
 			else
 				boutput(usr, "<span class='alert'>Redemption of \"[S.title]\" failed.</span>")
+////////navygreyhop////////
+/datum/achievementReward/ntso_commander
```

Comment:
I think you've overwritten the `ntso_commander` here, you probably need a unique name

Reply:
that would have been bad